notebooks/Emukit-tutorial-bayesian-optimization-context-variables.ipynb

```diff
@@ -0,0 +1,248 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bayesian optimization with context variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we are going to see how to used Emukit to solve optimization problems in which certain variables are fixed during the optimization phase. These are called context variables. For details see:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from emukit.test_functions import branin_function\n",
+    "from emukit.core import ParameterSpace, ContinuousParameter, DiscreteParameter\n",
+    "from emukit.experimental_design.model_free.random_design import RandomDesign\n",
+    "from GPy.models import GPRegression\n",
+    "from emukit.model_wrappers import GPyModelWrapper\n",
+    "from emukit.bayesian_optimization.acquisitions import ExpectedImprovement\n",
+    "from emukit.bayesian_optimization.loops import BayesianOptimizationLoop\n",
+    "from emukit.core.loop import FixedIterationsStoppingCondition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "f, _ = branin_function()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we define the domain of the function to optimize as usual."
```

Comment:
Can you refer here to a notebook that explains what you mean with 'as usual'? Someone might only come across this notebook and then it is hard to guess what is meant.

Reply:
removed the as usual as what we do is self explanatory.